biz.aQute.repository/bnd.bnd

```diff
@@ -20,8 +20,11 @@
 -testpath: \
 	${junit},\
 	${mockito},\
+	biz.aQute.junit;version=latest;packages=aQute.junit.rule,\
```

Comment:
Once you fix the export package above, the `packages` is not necessary since you export the right package.

Reply:
But I think this sort of violates the "rule" were _we_ do not depend upon `biz.aQute.junit` in any of our projects. We discussed this on a call with Peter and asserted it was the case the no Bnd project depends upon `biz.aQute.junit` for testing. This PR will change things, so perhaps we should not define such a rule here. Just get the classpath you need in the actual test case.

Reply:
right.. and I do remember this conversation. This position however leaves no place to add convenient junit rules to bnd. This particular rule has been laying around my branches for months. I've been moving around as needed in several branches because I can't find a home for it. Furthermore, I'd like to add even more rules.

Reply:
Make a separate project? I also have a runtime project for interesting bundles from the bnd world. A testutil project would be nice as well.

Reply:
> I've been moving around as needed in several branches because I can't find a home for it. Furthermore, I'd like to add even more rules.

Why not put the junit.rules package in the aQute.libg project? Then it would be easily accessible for our test cases from the -buildpath/-testpath. We don't need to put the package in a jar.